core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -149,4 +164,40 @@ class UserServiceSpec extends FlatSpecLike with TestDriverComponent with Mockito
       actual.asInstanceOf[RawlsExceptionWithErrorReport].errorReport.statusCode shouldEqual Option(StatusCodes.NotFound)
     }
   }
+
+  // 403 when workspace exists in this billing project to be deleted
+  it should "fail with a 400 when workspace exists in this billing project to be deleted" in {
+    withEmptyTestDatabase { dataSource: SlickDataSource =>
+      val project = defaultBillingProject
+      runAndWait(rawlsBillingProjectQuery.create(project))
+      runAndWait(workspaceQuery.save(workspace))
+
+      val mockSamDAO = mock[SamDAO]
+      when(mockSamDAO.userHasAction(SamResourceTypeNames.billingProject, project.projectName.value, SamBillingProjectActions.deleteBillingProject, userInfo)).thenReturn(Future.successful(true))
+
+      val userService = getUserService(dataSource, mockSamDAO)
+
+      val actual = userService.DeleteBillingProject(defaultBillingProjectName).failed.futureValue
+      assert(actual.isInstanceOf[RawlsExceptionWithErrorReport])
+      actual.asInstanceOf[RawlsExceptionWithErrorReport].errorReport.statusCode shouldEqual Option(StatusCodes.BadRequest)
```

Comment:
try this (and below too):
```
val actual = intercept[RawlsExceptionWithErrorReport] {
  userService.DeleteBillingProject(defaultBillingProjectName).futureValue
}
actual.errorReport.statusCode shouldEqual Option(StatusCodes.BadRequest)
```
intercept is a little cleaner for for expecting an exception

Reply:
Thanks!

Reply:
should start with Await.result(....